### PR TITLE
Pydicom live ebuild

### PR DESCRIPTION
**Pull request by TheChymera**
new live ebuild supporting more python versions

### CODE REVIEW

File: sci-libs/pydicom/pydicom-0.9.8.ebuild [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2015 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Header: $
 
 EAPI=5
 
-PYTHON_COMPAT=( python2_7 )
+PYTHON_COMPAT=( python2_6 python2_7 )
```
Comment (jlec) [inline sci-libs/pydicom/pydicom-0.9.8.ebuild]: No py2.6 support.

File: sci-libs/pydicom/pydicom-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_6 python2_7 python3_2 python3_3 python3_4 )
```
Comment (jlec) [inline sci-libs/pydicom/pydicom-9999.ebuild]: no py2.6 and py3.2 support

Comment (TheChymera) [inline sci-libs/pydicom/pydicom-9999.ebuild]: how do you know? I thought it lists all of them [here](https://github.com/darcymason/pydicom/blob/master/source/setup.py).

Comment (jlec) [inline sci-libs/pydicom/pydicom-9999.ebuild]: Gentoo doesn't support them, as the aren't support security wise by upstream.

**Comment by TheChymera** [comment]
@jlec perhaps this is ready?

File: sci-libs/pydicom/pydicom-9999.ebuild [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+inherit distutils-r1 git-r3
+
+DESCRIPTION="A pure python package for parsing DICOM files"
+HOMEPAGE="https://code.google.com/p/pydicom/"
+SRC_URI=""
+EGIT_REPO_URI="https://github.com/darcymason/pydicom"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```
Comment (jlec) [inline sci-libs/pydicom/pydicom-9999.ebuild]: No keywords in live ebuilds.

**Comment by TheChymera** [comment]
@jlec bump again?

File: sci-libs/pydicom/pydicom-9999.ebuild [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+# $Header: $
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 python3_3 python3_4 )
+
+inherit distutils-r1 git-r3
+
+DESCRIPTION="A pure python package for parsing DICOM files"
+HOMEPAGE="https://code.google.com/p/pydicom/"
```
Comment (jlec) [inline sci-libs/pydicom/pydicom-9999.ebuild]: google code will be shut down. so please fix the homepage.

**Comment by TheChymera** [comment]
any updates?

File: sci-libs/pydicom/ChangeLog [hunk 5]
```diff
@@ -1,8 +1,32 @@
...
+
+  31 Mar 2015;  <chymera@gentoo.org> +pydicom-9999.ebuild:
+  new live ebuild
+
+  31 Mar 2015;  <chymera@gentoo.org> pydicom-0.9.8.ebuild:
+  removed python 3 support (only inofficial:
+  https://dcmstack.readthedocs.org/en/latest/Introduction.html#dependencies )
+  added python 2.6 support
+
+  31 Mar 2015;  <chymera@gentoo.org> pydicom-0.9.8.ebuild:
```
Comment (marbre) [inline sci-libs/pydicom/ChangeLog]: E-Mail should be fixed, also the commits were merged.

**Pull request merged**